src/Symfony/Component/Form/ChoiceList/Factory/ChoiceListFactoryInterface.php

```diff
@@ -78,7 +78,7 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, $value = nul
      * attributes that should be added to the respective choice.
      *
      * @param array|callable|null $preferredChoices The preferred choices
-     * @param callable|null       $label            The callable generating the
+     * @param callable|bool|null  $label            The callable generating the
      *                                              choice labels
```

Comment:
btw, we should explain the `false` value here.

Reply:
Done. In the same way I did it here: https://github.com/symfony/symfony/pull/32935/files#diff-565fb1563b9eb5b65d8a69d08ed56b2dR35